### PR TITLE
cedar: project public types, constructors, and constants from the types package through the cedar package for convenience

### PR DESCRIPTION
**Pull request by patjakdev**
*Issue #, if available:* None

*Description of changes:* For convenience of use, this PR adds projections of all the public types, constructors, and constants from the `types` package through the `cedar` package.

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const (
+	True  = types.True
+	False = types.False
+)
+
+const (
+	DecimalPrecision = types.DecimalPrecision
+)
+
+// Constructors
```
Comment (philhassey) [inline types.go:42]: Check how this reads in pkgsite.  It might be nice to either copy the docs, or provide a [link] to the docs for each of the functions in the types package.

Comment (patjakdev) [reply types.go]: Good call. I've fleshed out the docs for the rest of the functions.

File: types/entities.go [hunk 2]
```diff
@@ -21,7 +21,10 @@ type Entity struct {
 }
 
 func (e Entities) MarshalJSON() ([]byte, error) {
-	s := e.toSlice()
+	s := maps.Values(e)
```
Comment (patjakdev) [inline types/entities.go:24]: Unrelated change. I just noticed that these could be inlined while I was looking for functions to export...

**Review by philhassey — APPROVED** [review]
LGTM
